tracer/src/Datadog.Trace/Ci/TestModule.cs

```diff
@@ -605,4 +635,75 @@ internal void RemoveSuite(string name)
             _suites.Remove(name);
         }
     }
+
+    internal bool EnableIpcClient()
```

Comment:
Any risk of concurrency here?

Reply:
I'm not worried about it, but added concurrency protection here: [4f5447d](https://github.com/DataDog/dd-trace-dotnet/pull/5551/commits/4f5447df2161e4da804c9f2184cecb53c8fb9079)

Reply:
If we're adding concurrency protection here, do we need to add it _everywhere_ we access _ipClient 🤔? If so, is it easier to use Interlocked accesses?